spec/controllers/v0/sessions_controller_spec.rb

```diff
@@ -552,6 +554,12 @@ def verify_session_cookie
               session: {
                 valid: false,
                 errors: ["Uuid can't be blank"]
+              },
+              identity: {
+                valid: false,
+                errors: ["Uuid can't be blank"],
+                authn_context: nil,
+                loa: { current: 1, highest: 1 }
```

Comment:
in the future, I would like for us to look at not having authn_context and loa in the JSON payload. The FE probably doesn't need to know this, or there are other ways of communicating this. This could get really confusing since LOA means different things internally and externally. The goal is to just get this stuff in Sentry for debugging purposes. I don't know that we would want it here as its just one more place that FE might form a dependency on something I would like to deprecate.

Reply:
Sure. This is just getting logged to sentry, not going to the front end, right?